include/color_spinor_field.h

```diff
@@ -493,6 +493,7 @@ namespace quda {
     size_t NormBytes() const { return norm_bytes; }
     size_t TotalBytes() const { return bytes + norm_bytes; }
     size_t GhostBytes() const { return ghost_bytes; }
+    size_t GhostFaceBytes(int i) const { return ghost_face_bytes[i]; }
```

Comment:
Looks like this should be squared with `0028fd019c65bdf200bbaf1ff832c56637cca185`

Reply:
https://github.com/lattice/quda/commit/0028fd019c65bdf200bbaf1ff832c56637cca185

Reply:
The addition of the local surfaces shouldn't really intersect with this.  That just adds some helpers to return the non-extended surface of a given extended field.  For a field that isn't extended, the local surface and regular surface are the same.

Reply:
Please reopen but for now I am marking this as resolved as above commit is from the gk branch.